build.gradle

```diff
@@ -30,4 +30,25 @@ test.finalizedBy("checkstyleMain")
 task runHelloWorldJoseG (dependsOn: 'classes', type: JavaExec) {
     main = 'jalau.at18.katas.helloworld.joseg.Main'
     classpath = sourceSets.main.runtimeClasspath
-}
\ No newline at end of file
+}
+
+task runKata1 (dependsOn: 'classes', type: JavaExec) {
```

Comment:
Hi @MariaAT18 , please add your name to your tasks so the do not tend to conflict with your teammates' tasks